pages/[type]/[key]/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PlaygroundApp from "../../../src/App";
+
+const data = require('../../../src/data.json');
+const dataKeys = Object.keys(data) ?? [];
+
+const RoutePage = ({ parsedData }: { parsedData: any }) => {
+  return (
+    <PlaygroundApp pageData={parsedData} />
+  );
+}
+
+function parseKey(value: string): string | undefined {
```

Comment:
this code is same as the one above, I think it should be extracted to utils or helpers or any folder with those purpose